<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>punctajeInsert</class>
 <widget class="QDialog" name="punctajeInsert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert Punctaje</string>
  </property>
  <widget class="QComboBox" name="nrMatricolBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr. matricol:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="anStudiuBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bonusEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="medieStudentEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nrCrediteEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>An de studiu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bonus:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Medie student:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr. credite:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okInsert">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelInsert">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
